SIG.md

```diff
@@ -0,0 +1,86 @@
+
+Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:
+
+- Policy
+- Scalability
+- Docs
+
+Each SIG must have a charter that specifies its scope (topics, subsystems, code repos, and/or directories), responsibilities, and areas of authority. See the SIG charter process for details on how charters are managed. SIGs are relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-project processes (e.g., the release process) and assets (e.g., the Cilium repo).
+
+SIGs are designed to help contributors gain depth in a specific area of the Cilium project. Committers that are aspiring reviewers of the Cilium project can gain expertise by getting involved with a SIG and getting assigned PRs.
+
+SIGs don't have any additional authority - they are expected to drive towards solutions that can be accepted by the project's committer community - ideally by consensus, and if necessary, a vote. 
+
+A primary reason that SIGs exist is as forums for collaboration. Much work in a SIG should stay local within that SIG. However, SIGs must communicate in the open, ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions, and periodically communicate a high-level summary of the SIG's work to the community.
+
+# SIG Roles
+
+## SIG Lead
+
+- Membership tracked in [sigs.yaml](./sigs.yaml)
+- MUST be a reviewer of one or more areas of the project, and should be a subject matter expert in their SIG area.
+- MUST ensure notes are made available.
+- MUST organize a method of keeping cadence (meetings, async channel in Cilium Slack, etc.), and make sure [sigs.yaml](./sigs.yaml) is up to date.
+- SHOULD facilitate cadence but MAY delegate to other members or future Leads/Leads in training.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MAY delegate communication and note taking to other contributors in the same SIG where appropriate.
+
+### Requirements
+Leads MUST be at least a "reviewer" on our [contributor ladder](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#reviewer) to be eligible to hold a leadership role within a SIG.
+SIGs MAY prefer various levels of domain knowledge depending on the role. This should be documented.
+
+### On-boarding and Off-boarding Leads
+Leads MAY decide to step down at anytime and propose a replacement. Use lazy consensus amongst the other Leads with fallback on majority vote to accept the proposal. The candidate SHOULD be supported by a majority of SIG contributors (active members of the Cilium organization engaged with creation and reviews in the scope of the SIG).
```

Comment:
I think we can drop this line? The following line covers the same information but with a conflicting approach.
```suggestion

```